.gitlab-ci.yml

```diff
@@ -189,11 +180,8 @@ Test:
   - npm run check-types
   - npm run test
 
-  only:
-  - /^nodeploy\/.*$/
-  except:
-  - dev
-  - master
+  rules:
+    - if: $CI_COMMIT_BRANCH != "master" && $CI_COMMIT_BRANCH != "dev" && $CI_COMMIT_BRANCH =~ /^nodeploy\/.*$/
```

Comment:
_Not a blocker, but:_

Aren't these two rules: `$CI_COMMIT_BRANCH != "master" && $CI_COMMIT_BRANCH != "dev" &&` redundant? Isn't the rule saying "only run this job for branches that start with `nodeploy/`"?

_(As an aside, we have almost never used `nodeploy` branches; I used them maybe a couple of times, and I am pretty sure no-one else did at all. We should probably review whether there is still any value in them)_